README.md

```diff
@@ -2,6 +2,8 @@
 
 [[![Join the chat at https://opentx.rocket.chat](https://camo.githubusercontent.com/3d659054abd6ce21c0e47cf3b83a51bda69ca282/68747470733a2f2f64656d6f2e726f636b65742e636861742f696d616765732f6a6f696e2d636861742e737667)](https://opentx.rocket.chat)]()
 
+Target Audience: Developers
```

Comment:
Do you realize that this file produces the content for the first page of the book? This file generates this page https://opentx.gitbooks.io/opentx-2-2-lua-reference-guide/content/

Perhaps I am not understanding what you are trying to do, so go ahead and add more content, perhaps then it will be more clear what you want to do in the end.

Thanks for helping.